common/journal_entries/dvg_south_africa_je.txt

```diff
@@ -0,0 +1,513 @@
+﻿je_road_to_rickusdorp = {
+	icon = "gfx/interface/icons/event_icons/event_map.dds"
+
+	complete = {
+		owns_entire_state_region = STATE_VRYSTAAT 
+		custom_tooltip = {
+		text = dvg_basuto_no_rickusdorp_tt
+			c:BST = {
```

Comment:
In journals also check if countries exist before doing something with them. What if the country does not exist anymore?

Reply:
For this entry it doesn't matter if Basuto exists it's just making sure that either opstalia annexed basuto or Leeuwin has also made progress on their starting content, since they also get events that encourage them to take the little bit of northern cape owned by basuto. It's a weird way to check the completion but i set it up like this to make sure that Opstalia and leeuwin share a border before moving on to their other missions